test/test_events.py

```diff
@@ -274,19 +294,26 @@ def test_events_normal(uqcsbot: MockUQCSBot):
     uqcsbot.post_message(TEST_CHANNEL_ID, "!events")
     messages = uqcsbot.test_messages.get(TEST_CHANNEL_ID, [])
     assert len(messages) == 2
-    expected = ("Events in the *next _2_ weeks*\r\n"
-                "*MAY 28 12:00 - 13:00* - `<https://www.itee.uq.edu.au/introduction"
-                + "-functional-programming|Introduction to functional programming"
-                + " - Tony Morris, Software Engineer at Data61>` - _78-420_\r\n"
-                "*MAY 29 13:00 - 14:00* - `<https://www.itee.uq.edu.au/performance-enhancement-"
-                + "software-defined-cellular-5g-and-internet-things-networks"
-                + "|Performance Enhancement of Software Defined Cellular 5G &amp;"
-                + " Internet-of-Things Networks - Furqan Khan>` - _78-430_\r\n"
-                "*AUG 2 0:00 - AUG 5 23:59* - `CodeNetwork Hackathon` - _River City Labs_\r\n"
-                "*AUG 2 18:00 - 20:00* - `Dr Corey Shou` - _TBC_\r\n"
-                "*AUG 6 18:00 - 20:00* - `vim &amp; tmux - Neil Ashford` - _Hawken 50-T103_\r\n"
-                "*AUG 13 18:00 - 20:00* - `Robogals x UQ Robotics x UQCS Social Event`"
-                + " - _Hawken 50-C207_")
+    expected = (
+        "Events in the *next _2_ weeks*\r\n"
+        "*TUE MAY 28 12:00 - 13:00* - "
+        "`<https://www.itee.uq.edu.au/introduction"
+        + "-functional-programming|Introduction to functional programming"
+        + " - Tony Morris, Software Engineer at Data61>` - _78-420_\r\n"
+        "*WED MAY 29 13:00 - 14:00* - "
+        "`<https://www.itee.uq.edu.au/performance-enhancement-"
+        + "software-defined-cellular-5g-and-internet-things-networks"
+        + "|Performance Enhancement of Software Defined Cellular 5G &amp;"
+        + " Internet-of-Things Networks - Furqan Khan>` - _78-430_\r\n"
+        "*FRI AUG 2 0:00 - MON AUG 5 23:59* - `CodeNetwork Hackathon` - "
+        "_River City Labs_\r\n"
+        "*FRI AUG 2 18:00 - 20:00* - `Dr Corey Shou` - _TBC_\r\n"
+        "*TUE AUG 6 18:00 - 20:00* - `vim &amp; tmux - Neil Ashford` - "
+        "_Hawken 50-T103_\r\n"
```

Comment:
Message line continued; begin with `+`.
Move joining characters (` - `) to latter line.